src/main.sh

```diff
@@ -4,9 +4,12 @@ set -e
 
 function _check_setup {
   # Checking git and secret-plugin setup:
-  if [[ ! -d ".git" ]] || [[ ! -d ".git/hooks" ]]; then
+  set +e
```

Comment:
We use `set -e` in all our code, I guess there's no reason to break the rule here.